### DISCUSSION TITLE
[Dashboard]: Multi-pane section?

### DISCUSSION THREAD

**Discussion by OmegaLambda1998** [folke/snacks.nvim#169] [Ideas]
Hi, is it possible to have a section (like the header) span across multiple panes?
I pretty sure I can hack something together by first pre-calculating the header text, then splitting it in half and assigning each half to its own text plane, but I wanted to check first before wasting time on it

The end result would be a header which takes the full with of two panes, followed by two columns of information

**Comment by CodeBoyPhilo** [comment]
Hi, it is definitely possible to have one section _appears_ to span across multiple pane. Instead of splitting one section into two, one dumb way to do it is to add dummy sections and adjust the indent and paddings. 

My original dashboard has everything in one pane:
![Screenshot 2024-12-05 at 10 17 16 PM](https://github.com/user-attachments/assets/ac64ffd3-849c-49a1-8805-e2b79fcaa81f)

By adding the dummy section:
```lua
{section="terminal", cmd="echo ' '", padding=1}
```
to match up with the _CodeBoyPhilo.nvim_ chafa image and with some adjustments to indentation, we can have the first chafa image to span across two panes.
Meanwhile, the second chafa image, the recent_files and projects stay in two separate panes nicely.
Next, we have the startup and colored texts appear to locate in the middle.
We can easily achieve this by directly manipulating the indent without adding dummy sections.
This method is really naive and takes trials and errors to make everything look right.
It's not a good way to achieve the desired effects and it may only work in limited scenarios, but it worths exploration first without taking the efforts to do the precise calculations.
![Screenshot 2024-12-05 at 10 12 01 PM](https://github.com/user-attachments/assets/1d3de5be-5fa0-4ed6-b916-b6f1477533c1)
